apps/docs/stories/components/checkout.stories.tsx

```diff
@@ -127,7 +131,22 @@ const meta: Meta = {
       component += ` disable-payment-method-group`;
     }
 
-    component += `></justifi-checkout>`;
+    component += `>`;
+
+    if (args.withInsurance) {
+      console.log('inserting insurance slot')
```

Comment:
Do you plan to keep this console log or was this for testing purposes and left over? 🤔